packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -1,7 +1,9 @@
+// @ts-check
 const fs = require(`fs`)
 const path = require(`path`)
 const crypto = require(`crypto`)
 
+const _ = require(`lodash`)
```

Comment:
Are we using the compile step to transform this? Or should we manually change this to the named import version?